packages/@react-aria/combobox/docs/useComboBox.mdx

```diff
@@ -502,7 +502,6 @@ function Example() {
       items={filteredItems}
       inputValue={filterValue}
       onInputChange={setFilterValue}
-      placeholder="Enter recipient email"
```

Comment:
Did we have a request to remove it from aria-hooks docs? We could leave it in I think, unless we're trying to encourage people to make their own help texts? in which case we should probably have that in the examples which would make the example longer... so I don't know, discussion time.

Reply:
talked over with team, we're removing them in the aria examples as well because several browsers don't have enough contrast